service/gradle/jib.gradle

```diff
@@ -1,8 +1,9 @@
 jib {
     from {
-        image = "adoptopenjdk:11-jre-hotspot"
+        // Google's distroless images are openjdk, this is the simplest with adoptopenjdk
+        image = 'us.gcr.io/broad-dsp-gcr-public/base/jre:17-debian'
```

Comment:
Curious about the Broad dependency here -- is there a public image that could be better to use instead?

Reply:
I copied this from Verily repos since they were already vetted by security (atleast broad). 
here is a list of official images - https://github.com/docker-library/docs/blob/master/eclipse-temurin/README.md#supported-tags-and-respective-dockerfile-links
Is there a recommended one for Tanagra